portfolio/src/main/webapp/script.js

```diff
@@ -24,3 +24,27 @@ function closeNav() {
 }
 
 
+function getMessage() {
+  fetch('/data').then(response => response.text()).then((message) => {
+    document.getElementById('message-container').innerText = message;
+  });
+}
+
+
+function getMessageJSON() {
+  fetch('/data').then(response => response.json()).then((message) => {
+    const messageListElement.innerHTML = '';
+    messageListElement.appendChild(
+    createListElement('Message 1: ' + message[0]));
```

Comment:
Instead of depending explicitly on the response having 3 elements, can you use a looping construct to have it work properly for any size response?